### ISSUE TITLE
Process finished with exit code -1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by fanyu546250151555481656** [TAMU-VITA/GLNet#10]
This problem occurred when I ran this code on Windows: 
“Process finished with exit code -1073741819 (0xC0000005)”.
But this code can run if I delete "loss.backward()".Has anyone ever had this problem?

**Comment by chenwydj** [comment]
Sorry I did not try to train and evaluate our model on the Windows platform. Maybe you could try the evaluation process (see instructions in Readme).
